tests/integration/test_request_signing.py

```diff
@@ -0,0 +1,45 @@
+import json
+from sinch.core.signature import Signature
+
+
+def test_request_signature(
+    sinch_client_sync
+):
+    signature = Signature(
+        sinch_client_sync,
+        http_method="POST",
+        request_data=json.dumps({"test": "test"}),
+        request_uri="/verification/v1/verifications"
+    )
+    signature.calculate()
+    assert signature.authorization_signature
```

Comment:
testing against expected an expected value will ensure better tests against regression

Reply:
fixtures and assertions already added